spec/lib/pdf_fill/hash_converter_spec.rb

```diff
@@ -234,7 +234,7 @@ def call_set_custom_value(value, *args)
 
     it 'converts the hash correctly' do
       expect(
-        described_class.new('test', '%m/%d/%Y').transform_data(
+        described_class.new('test', '%m/%d/%Y', false, 8).transform_data(
```

Comment:
I think the two changes in this file are also unnecessary -- everything should work with the default arguments, and nothing in HashConverter is currently affected.